src/gui/session.py

```diff
@@ -22,5 +24,27 @@ def unload(self):
         del self.action
 
     def run(self):
-        QMessageBox.information(None, "Minimal plugin",
-                                "Do something Fred here")
+        if self.session.IsDataBaseLayerExistsInCurrentProject():
```

Comment:
A layer is not necessary opened so we should use a bool attribute member to detect if a database is currently opened.

Reply:
With the 5th commit of the branch, the button is only use to memorize the directory path and, if it doesn't exist, create the database.